napalm/ios/ios.py

```diff
@@ -128,6 +128,8 @@ def __init__(self, hostname, username, password, timeout=60, optional_args=None)
 
         self.profile = ["ios"]
 
+        self.set_canonical_interface = optional_args.get('canonical_int', False)
```

Comment:
How about `self.use_canonical_int`? I don't like `set_canonical_interface` as it makes me think we are doing a setter on an attribute.